modules/client-configuration/pages/supported-features-ubuntu.adoc

```diff
@@ -28,7 +30,7 @@ The icons in this table indicate:
 | Operating system packages | icon:times[role="danger"] | icon:times[role="danger"] | icon:times[role="danger"]
 | Registration | ``Salt`` | ``Salt`` | ``Salt``
 | Install packages | ``Salt`` | ``Salt`` | ``Salt``
-| Apply patches | icon:question[role="gray"] | icon:question[role="gray"] | icon:question[role="gray"]
```

Comment:
Not sure what the character before `Apply` was, but it wasn't a whitespace:

![image](https://user-images.githubusercontent.com/4226070/81422360-26f31780-9153-11ea-8579-fac4a3d40227.png)